lib/checkVerificationCode.ts

```diff
@@ -0,0 +1,32 @@
+import { FetchResult } from "apollo-link";
```

Comment:
What about using CQRS inside the `lib` directory for the future? Separating queries and commands that way?

Reply:
I agree, but I'm not sure about where to put what thought in the case of GraphQL. Am I right to think that queries goes into queries, and mutations/subscriptions are commands?

Reply:
![](https://steamuserimages-a.akamaihd.net/ugc/1473028176941850904/83F12B5612084AD2FBC9037113AFA6C80897045F/)